frontend/src/hooks/api/organization/queries.tsx

```diff
@@ -29,3 +30,26 @@ export const useRenameOrg = () => {
     }
   });
 };
+
+export const useCreateOrganization = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation<NewOrganizationResponse, {}, CreateNewOrganization>({
```

Comment:
As mentioned in another comment, just add `DTO` to it to make the naming convention similar.

Same goes for the `DeleteOrganization` type in `useDeleteOrganization`